cmd/lightchain/init.go

```diff
@@ -75,6 +76,13 @@ func newNodeCfgFromCmd(cmd *cobra.Command) (node.Config, setup.Network, error) {
 	shouldTrace, _ := cmd.Flags().GetBool(TraceFlag.Name)
 	traceLogFilePath, _ := cmd.Flags().GetString(TraceLogFlag.Name)
 
+	// This should be done inside of the `node.Init()` pkg but due to bad design,
```

Comment:
Would you mind to create an issue and include the number as part of this doc?

Reply:
done https://github.com/lightstreams-network/lightchain/pull/88/commits/802bf83493a760dae603bcba3875780755e846ac

Reply:
done https://github.com/lightstreams-network/lightchain/pull/88/commits/802bf83493a760dae603bcba3875780755e846ac